framework/src/components/system/system.js

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+const os = require('os');
+const crypto = require('crypto');
+const semver = require('semver');
+
+/**
+ * Main system methods. Initializes library with scope content and headers:
+ * - os
+ * - version
+ * - wsPort
+ * - httpPort
+ * - minVersion
+ * - protocolVersion
+ * - height
+ * - nethash
+ * - broadhash
+ * - nonce
+ *
+ * @class
+ * @requires crypto
+ * @requires os
+ * @requires semver
+ * @param {Object} options - System options
+ * @param {Object} logger
+ */
+class System {
+	constructor(config, logger, storage) {
+		this.logger = logger;
+		this.storage = storage;
+		this.headers = {
+			os: os.platform() + os.release(),
+			version: config.version,
+			wsPort: config.wsPort,
+			httpPort: config.httpPort,
+			minVersion: config.minVersion,
+			protocolVersion: config.protocolVersion,
+			height: 1,
+			nethash: config.nethash,
+			broadhash: config.nethash,
```

Comment:
is this some logical error or we really assigned `broadhash = nethash` ?

Reply:
Yes, It was like that.